corehq/ex-submodules/auditcare/utils/show_urls.py

```diff
@@ -1,5 +1,6 @@
 # from http://code.activestate.com/recipes/576974-show-all-url-patterns-in-django/
 # prints a tree of all the urls in the project
+from __future__ import print_function
```

Comment:
@dimagi/team-commcare-hq do this if you want to use `print(...)` and don't want @stickler-ci to complain about it.